### PR TITLE
add pytexit

### PR DESCRIPTION
**Pull request by Krande**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Krande** [comment]
@conda-forge/help-python, ready for review!

File: recipes/pytexit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/erwanp/pytexit/archive/{{ version }}.tar.gz
+  sha256: 3fa9f5010ee212bef1b0cb0e2b8383473b7f76144a343e97140fb751e463e7dd
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps -vv
```
Comment (ocefpaf) [inline recipes/pytexit/meta.yaml:15]: Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information. 

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/pytexit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - python >=3.8
+  run:
+    - python >=3.8
+    - six
+test:
+  source_files:
+    - pytexit/test
+  requires:
```
Comment (ocefpaf) [inline recipes/pytexit/meta.yaml:26]:
```suggestion
  requires:
    - pip
```

File: recipes/pytexit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - pytest
+    - pytest-cov
+  commands:
```
Comment (ocefpaf) [inline recipes/pytexit/meta.yaml:29]:
```suggestion
  commands:
    - pip check
```

**Pull request merged**